app/assets/javascripts/app/app.js

```diff
@@ -155,6 +156,30 @@ var app = {
 
   _changeLocation: function(href) {
     window.location.assign(href);
+  },
+
+  setupReport: function() {
+    console.debug("Loading report")
+    document.getElementById("report-content-form").addEventListener("submit", function(ev) {
+      if (ev) {  ev.preventDefault(); }
+      let form = ev.currentTarget;
+      $("#reportModal").modal("hide");
+      let textarea = document.getElementById("report-reason-field");
+      let report = {
+        item_id: form.dataset.reportId,
```

Comment:
Identifier 'item_id' is not in camel case.

Reply:
I'm not doing this refactor in this MR.